_rules/html-page-lang-b5c3f8.md

```diff
@@ -38,7 +38,7 @@ Each test target has a `lang` attribute that is neither empty ("") nor only [ASC
 
 ## Assumptions
 
-This rule assumes that the presence of a `lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.
+The language of the page can be set by other methods than the `lang` attribute, for example using HTTP headers or the `meta` element. These methods are not supported by all assistive technologies. This rule assumes that these other methods are insufficient to satisfying [Success Criterion 3.1.1: Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page).
```

Comment:
Still, these aren't known to be supported by **any** AT. Saying "are not supported by all assistive technologies" implies that it can be used. If we can't name a single one that supports it, we should be clear about that. Otherwise this should be an accessibility support note.

Reply:
According to tests done by @dd8 in https://github.com/act-rules/act-rules.github.io/issues/668#issuecomment-534606384 there methods are exposed to the accessibility tree by **some** browsers, so it will be available to **some** combinations of UA+AT.